src/services/apiService.ts

```diff
@@ -3,11 +3,12 @@ import { Auth } from "aws-amplify";
 
 export class ApiService {
   axiosInstance: AxiosInstance;
+  urlPath: string = "";
   traineeTisId = 123;
 
-  constructor() {
+  constructor(urlPath: string) {
+    this.urlPath = urlPath;
     this.axiosInstance = axios.create({
```

Comment:
Why not set the baseUrl here?

Reply:
I thought baseUrl needs to be absolute. Relative path is getting added at the trail of the base url.
Will modify it.